### PR TITLE
Fix ubuntu-22.04 runner issue

### PR DESCRIPTION
**Pull request by PatKamin**
Related issue: https://github.com/actions/runner-images/issues/9491

### CODE REVIEW

File: .github/workflows/sanitizers.yml [hunk 1]
```diff
@@ -34,6 +34,10 @@ jobs:
         apt-get update
         apt-get install -y cmake libnuma-dev libjemalloc-dev libtbb-dev libhwloc-dev
 
+    # Remove this once https://github.com/actions/runner-images/issues/9491 was fixed
+    - name: Decrease ASLR entropy due to the issue
+      run: sudo sysctl vm.mmap_rnd_bits=28
```
Comment (lukaszstolarczuk) [inline .github/workflows/sanitizers.yml:39]: no need for sudo 😉

**Review by bratpiorka — APPROVED** [review]

**Review by lukaszstolarczuk — COMMENTED** [review]
can you pls add link to the issue in the commit message

**Comment by PatKamin** [comment]
> can you pls add link to the issue in the commit message

Done

File: .github/workflows/sanitizers.yml (hunk 1 above)
Comment (PatKamin) [reply .github/workflows/sanitizers.yml]: Right, thanks.

**Review by lukaszstolarczuk — APPROVED** [review]
hope this builds :wink:

**Review by KFilipek — APPROVED** [review]
Fully agree

**Pull request merged**